app/src/main/java/com/github/kulebin/myfoursquareapp/http/IInterceptor.java

```diff
@@ -0,0 +1,16 @@
+package com.github.kulebin.myfoursquareapp.http;
+
+public interface IInterceptor {
+
+    <T> T onPreExecute(T request);
+
+    void onRequestExecuted(int statusCode, String url);
+
+    final class Impl {
+
+        public static IInterceptor newInstance() {
+            return new Interceptor();
```

Comment:
no default realization in http package. please create realization in App Class. It's ok and enough for now